Assets/Scripts/CupFall.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CupFall : MonoBehaviour
```

Comment:
I think we could make this more general, or you could even just use the `Animator` component.

Reply:
Yeah I agree, I thought the same, but I just wanted to get something done in time for GDDC. 

I was thinking of generalizing it in a later PR, but yeah might as well do it now. I still need to look into the Animator component more, likely in a later PR though.